README.md

```diff
@@ -10,6 +10,15 @@
 `pynxtools` is a tool designed for making your experimental data FAIR.
 It allows to develop ontologies and to create ontological instances based on the [NeXus format](https://www.nexusformat.org/).
 
+# Scope
+
+`pynxtools` (previously called `nexusutils`) is intended as a parser for combining various instrument output formats and electronic lab notebook (ELN) formats to an hdf5 file according to NeXus application definitions.
+
+Additionally, the software can be used as a plugin in the research data management system NOMAD for
```

Comment:
Hm, it adds the parser + the nexus metainfo. I would just leave it as it is and then later add docs for how to use it in NOMAD and how it will be available in NOMAD then.